staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -233,6 +233,18 @@ type Config struct {
 
 	// StorageVersionManager holds the storage versions of the API resources installed by this server.
 	StorageVersionManager storageversion.Manager
+
+	// DelayServerShutdown dictates when to initiate shutdown of the HTTP Server
+	// during the graceful termination of the apiserver.
+	// If true, we wait for existing requests in flight to be completed and
+	// then initiate a shutdown of the HTTP Server.
+	// If false, we initiate a shutdown of the HTTP Server
+	// as soon as ShutdownDelayDuration has elapsed.
+	DelayServerShutdown bool
```

Comment:
Can we invert the logic in the var name and the flag? `keepListeningDuringGracefulTermination`.